WordPress/src/main/java/org/wordpress/android/ui/prefs/MyProfileActivity.java

```diff
@@ -135,12 +187,18 @@ private void updateLabel(WPTextView textView, String text) {
                                                        final String hint,
                                                        final WPTextView textView,
                                                        final boolean isMultiline) {
+        mDialogTitle = dialogTitle;
+        mHint = hint;
+        mTextView = textView;
+        mIsMultiline = isMultiline;
         return new View.OnClickListener() {
             @Override
             public void onClick(View v) {
+                mIsDialogOpen = true;
+                mDialogText = textView.getText().toString();
                 DialogUtils.showMyProfileDialog(MyProfileActivity.this,
                         dialogTitle,
-                        textView.getText().toString(),
+                        mDialogText,
                         hint,
                         isMultiline,
                         new DialogUtils.Callback() {
```

Comment:
I see the same code being used here: https://github.com/wordpress-mobile/WordPress-Android/pull/3613/files#diff-920ac70fdbaa0671702545c12fc783e1R102
Can we create a new private class that implements `DialogUtils.Callback` and use it in both places?